tests/test_tax_rates.py

```diff
@@ -29,18 +29,18 @@ class TestTaxRateDecimal:
     @pytest.mark.parametrize(
         "inputted,expected",
         [
-            (Decimal("1"), Decimal("1.00")),
```

Comment:
Please add the following 3 cases as well:

- `percentage` is `100` 
- `percentage` is `decimal("100")` 
- `percentage` is `"100"`